desktop/apps/article/queries/relatedArticles.js

```diff
@@ -1,4 +1,4 @@
-export default `
+export const relatedArticles = `
   relatedArticlesPanel {
     slug
     thumbnail_title
```

Comment:
@kanaabe - I noticed that this was throwing strange errors; it looks like the template literal needs a name

Reply:
Just out of curiosity, what tooling wants this to have a name?

Reply:
Looking now, it seemed like 
```
export default ` ... `
``` 

was a transient issue -- trying now, everything is 👍

Reply:
Okeydokey. I’m not actually in favour of anonymous exports btw ;)